client/src/components/J40Footer.tsx

```diff
@@ -1,17 +1,15 @@
 import React from 'react';
 import {
   Address,
-  Footer,
```

Comment:
I think I'd like to do this as part of a different PR. We need to update a bunch of packages and I could do it at the same time. What do you think?

Reply:
Which other packages are you thinking require updating?

Reply:
```
> npm outdated    
Package                           Current   Wanted   Latest  Location
@trussworks/react-uswds             2.0.0      git      git  justice40-tool
@formatjs/cli                      4.2.30   4.2.32   4.2.32  justice40-tool
@testing-library/cypress            7.0.7    7.0.7    8.0.0  justice40-tool
@testing-library/react             11.2.7   11.2.7   12.0.0  justice40-tool
@types/jest                       26.0.24  26.0.24   27.0.1  justice40-tool
@types/node                       15.14.7  15.14.9   16.7.1  justice40-tool
@types/react                      17.0.16  17.0.19  17.0.19  justice40-tool
@typescript-eslint/eslint-plugin   4.29.1   4.29.3   4.29.3  justice40-tool
@typescript-eslint/parser          4.29.1   4.29.3   4.29.3  justice40-tool
babel-preset-gatsby                1.11.0   1.12.0   1.12.0  justice40-tool
cypress                             7.7.0    7.7.0    8.3.0  justice40-tool
cypress-axe                        0.12.2   0.12.2   0.13.0  justice40-tool
eslint-plugin-formatjs             2.17.3   2.17.5   2.17.5  justice40-tool
gatsby                             3.11.1   3.12.0   3.12.0  justice40-tool
gatsby-cli                         3.11.0   3.12.0   3.12.0  justice40-tool
gatsby-plugin-react-helmet         4.11.0   4.12.0   4.12.0  justice40-tool
gatsby-plugin-sass                 4.11.0   4.12.0   4.12.0  justice40-tool
mapbox-gl                          1.13.1   1.13.1    2.4.1  justice40-tool
react-intl                         5.20.7  5.20.10  5.20.10  justice40-tool
sass                               1.37.5   1.38.0   1.38.0  justice40-tool
sass-loader                        11.1.1   11.1.1   12.1.0  justice40-tool
ts-jest                            27.0.4   27.0.5   27.0.5  justice40-tool
uswds                              2.12.0   2.12.1   2.12.1  justice40-tool
```

Reply:
IMHO, I think this upgrading the libs are important to do, it's also major overhaul and a major risk. I commented in the other ticket (in case you may not have seen it) that testing (unit and e2e) should be in place before upgrading any library (ideally).

In our case, we have identified one particular component of one particular library that could be removed as the fork had to do with the window object of the footer.  It's my opinion that removing this one library now is better than spending a larger effort updating all libraries. It has a few benefits (if it works/ builds/ runs): 

1. We can potentially remove the dependency on having users have to start with app with `yarn`
2. We may be able to remove the hack in the docker file that has to engage in network-concurrency
3. Both of the above help in the open-sourceness of the app, which is the goal for this sprint

I believe pushing this off as part of a larger scope of upgrading all libraries will most likely not happen as attempting to upgrade all the aforementioned libraries will take a good deal of effort. Postponing it will then not allow the realization of the above 3 which could be easy lift.

How does all that sit with you?

Reply:
This PR only changes Trussworks from custom branch to `^2.0.0`